test/fixtures/parser/mysql/create_table.yml

```diff
@@ -1,43 +1,43 @@
-file:

```

Comment:
I'm not sure why this file shows as "changed". I wonder if the line endings changed?